src/services/Store/store/persist.config.spec.tsx

```diff
@@ -0,0 +1,17 @@
+import { defaultContacts } from '@database';
+import { DataStore, LSKeys } from '@types';
+import { values } from '@vendor';
+
+import { migrations } from './persist.config';
+
+describe('Persist Migrations', () => {
+  it('3: Updates the MYC_DONATE_ADDRESS in a users persistence layer', () => {
+    const migration = migrations['3'];
+    const contact = values(defaultContacts)[0];
+    const actual = migration({
+      [LSKeys.ADDRESS_BOOK]: [{ ...contact, label: 'Deprecated label' }]
```

Comment:
Can we test that this keeps other entries in the address book, just as a sanity check?

Reply:
Updated.